src/pages/home/report/ReportActionItem.tsx

```diff
@@ -555,7 +574,7 @@ function ReportActionItem({
                 </ShowContextMenuContext.Provider>
             );
         } else if (action.actionName === CONST.REPORT.ACTIONS.TYPE.REIMBURSEMENT_QUEUED) {
-            const linkedReport = ReportUtils.isChatThread(report) ? ReportUtils.getReport(report.parentReportID) : report;
+            const linkedReport = ReportUtils.isChatThread(report) ? parentReport : report;
```

Comment:
Looks we can get ride of the `parentReport`. We only need 2 props from the parent
- reportID which can be replaced by report.parentReportID
- ownerAccountID write an util function for that